README.md

```diff
@@ -31,22 +51,21 @@ License at
 
     http://www.apache.org/licenses/LICENSE-2.0
 
-Unless required by applicable law or agreed to in writing, software distributed
-under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
-CONDITIONS OF ANY KIND, either express or implied. See the License for the
-specific language governing permissions and limitations under the License.
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
 
 <!-- refs -->
 [ci]: https://travis-ci.org/runconduit/conduit
 [ci-badge]: https://travis-ci.org/runconduit/conduit.svg?branch=master
 [coc]: https://github.com/linkerd/linkerd/wiki/Linkerd-code-of-conduct
+[conduit-demo]: https://conduit.io/getting-started/#install-the-demo-app
+[conduit-docs]: https://conduit.io/docs/
 <!-- [examples]: https://github.com/runconduit/conduit-examples -->
 [grpc]: https://grpc.io/
 [l5d]: https://linkerd.io/
 [license-badge]: https://img.shields.io/github/license/linkerd/linkerd.svg
 [logo]: https://user-images.githubusercontent.com/240738/33589722-649152de-d92f-11e7-843a-b078ac889a39.png
 <!-- [releases]: https://github.com/runconduit/conduit -->
 [rust]: https://rust-lang.org/
-[twitter]: https://twitter.com/runconduit/
+[Twitter]: https://twitter.com/runconduit/
```

Comment:
I'd prefer to leave this lower-cased. These anchors are just references -- they don't get rendered in the text of the page. And the convention for the rest of the anchors is all lower case.